CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Deprecated: `-e` and `--extract` CLI flags, and the `extractStyleTagsFromHtml` node API option. If you use these flags or option, please consider creating a processor for the community. See the [release planning](/docs/user-guide/release-planning.md) document for more details.
 - Added: `ignoreProperties: []` option for `declaration-block-no-duplicate-properties`.
+- Fixed: accuracy of warning positions are blocks without brackets when using SugarSS parser.
```

Comment:
How about "accuracy of warning positions for empty blocks when using SugarSS parser"